common/pc/display/hidpi.nix

```diff
@@ -0,0 +1,13 @@
+# Shared configuration which most (if not all) HiDPI setups should have.
+{ config, lib, pkgs, ... }:
+
+{
+  services.xserver.dpi = lib.mkDefault 196;
+  fonts.fontconfig.dpi = lib.mkDefault 196;
+
+  # Fix sizes of GTK/GNOME ui elements
+  environment.variables = {
+    GDK_SCALE = lib.mkDefault "2";
+    GDK_DPI_SCALE= lib.mkDefault "0.5";
```

Comment:
How about this?

```
{ pkgs, lib, ... }: {
  console.font = lib.mkDefault
    "${pkgs.terminus_font}/share/consolefonts/ter-u28n.psf.gz";
  console.earlySetup = lib.mkDefault true;
  boot.loader.systemd-boot.consoleMode = lib.mkDefault "max";

  services.xserver.dpi = lib.mkDefault 196;
  fonts.fontconfig.dpi = lib.mkDefault 196;
  environment.variables = lib.mkDefault {
    GDK_SCALE = "2";
    GDK_DPI_SCALE = "0.5";
    _JAVA_OPTIONS = "-Dsun.java2d.uiScale=2";
  };
}
```

Reply:
This adds the linux consol, java and systemd-boot.

Reply:
@Mic92 sweet, I'll append that.

Reply:
@Mic92, I made the changes that you suggested, not had time to test it yet, but the tests are passing.

Thank you for the feedback!